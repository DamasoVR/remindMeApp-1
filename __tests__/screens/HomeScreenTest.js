import 'react-native';
import React from 'react';
import HomeScreen from '../../src/screens/HomeScreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

let component;
const navigationMock = { navigate: jest.fn() };

describe('Home Screen - ', () => {

  it('should render correctly', () => {
    component = renderer.create(
      <HomeScreen navigation={navigationMock}/>
    ).toJSON();
    expect(component).toMatchSnapshot();
  });

});
