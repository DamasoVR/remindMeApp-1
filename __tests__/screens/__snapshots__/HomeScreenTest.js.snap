// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen -  should render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "rgb(237,24,74)",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
>
  <Image
    source={
      Object {
        "process": [Function],
      }
    }
    style={
      Object {
        "marginTop": 80,
      }
    }
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderColor": "#ffffff",
        "borderRadius": 27,
        "borderStyle": "solid",
        "borderWidth": 2,
        "height": 55,
        "paddingTop": 15,
        "width": 325,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "white",
          "fontFamily": "Avenir",
          "fontSize": 17,
          "textAlign": "center",
        }
      }
    >
      COMENZAR
    </Text>
  </View>
  <Image
    source={
      Object {
        "process": [Function],
      }
    }
  />
</View>
`;
